mailer.js

```diff
@@ -219,12 +253,15 @@ module.exports = function (log) {
       templateValues: {
         device: this._formatUserAgentInfo(message),
         email: message.email,
+        ip: message.ip,
         link: link,
+        location: gettext(this._constructLocationString(message.location)),
```

Comment:
Oh, I see this tries to localize the *whole* location string here.  I'm pretty sure this won't work out, as gettext can only be called with static(ish) strings rather than something built at runtime like this.  I think you'll need to split this up into a dynamic `location` string built at runtime, and a static `"(estimated)"` or similar that can be handled by the translation machinery.